src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,117 @@
+                manufacturer.setName(resultSet.getString(2));
+                manufacturer.setCountry(resultSet.getString(3));
+                return Optional.of(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacturer from DB, id: " + id, e);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String getAllManufacturersQuery = "SELECT * FROM manufacturers WHERE is_deleted = FALSE";
+        List<Manufacturer> allManufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getAllManufacturers = connection
+                         .prepareStatement(getAllManufacturersQuery)) {
+            ResultSet resultSet = getAllManufacturers
+                    .executeQuery();
+            while (resultSet.next()) {
+                Long id = resultSet.getObject("id", Long.class);
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                Manufacturer manufacturer = new Manufacturer();
+                manufacturer.setId(id);
+                manufacturer.setName(name);
+                manufacturer.setCountry(country);
+                allManufacturers.add(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all manufacturers from DB", e);
+        }
+        return allManufacturers;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateManufacturerQuery = "UPDATE manufacturers "
+                + "SET name = ?, country = ? WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getAllManufacturers = connection
+                         .prepareStatement(updateManufacturerQuery)) {
+            getAllManufacturers.setString(1, manufacturer.getName());
+            getAllManufacturers.setString(2, manufacturer.getCountry());
+            getAllManufacturers.setObject(3, manufacturer.getId());
+            getAllManufacturers.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update manufacturer from DB, id: "
+                    + manufacturer.getId(), e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteManufacturerQuery = "UPDATE manufacturers SET is_deleted = true WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement deleteManufacturer = connection
+                         .prepareStatement(deleteManufacturerQuery)) {
+            deleteManufacturer.setObject(1, id);
+            return deleteManufacturer.executeUpdate() >= 1;
```

Comment:
```suggestion
            return deleteManufacturer.executeUpdate() > 0;
```
better compare with 0

Reply:
> better compare with 0

fixed)